serverless/serverless-getting-started.adoc

```diff
@@ -17,16 +17,17 @@ Developers on {ServerlessProductName} can use the provided Kubernetes native API
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
-[id="serverless-supported-configs_{context}"]
+[id="serverless-getting-started-supported-configs_{context}"]
 == Supported Configurations
 
 The set of supported features, configurations, and integrations for {ServerlessProductName} (current and past versions) are available at the link:https://access.redhat.com/articles/4912821[Supported Configurations page].
 
-include::modules/serverless-components.adoc[leveloffset=+1]
-
 :FeatureName: Knative Eventing
 include::modules/technology-preview.adoc[leveloffset=+2]
 
+[id="serverless-getting-started-next-steps_{context}"]
```

Comment:
@abrennan89, there is no `context` for assembly anchor IDs; That's just for modules so a module can be included anywhere and have a unique ID.